### PR TITLE
added function to create member using type

### PR DESCRIPTION
**Pull request by javidang**

### CODE REVIEW

**Comment by javidang** [comment]
@nikanka-token , after @yuhanniu 's comment on PR, I decided to not expose tppSdk and let it to create member of any type, but tpp type. So during creating member, you can use number to choose which type of tpp you want. Currently I only implemented type 2. Let me know if that part of logic is wrong.

File: tpp/src/main/TokenClient.js [hunk 1]
```diff
@@ -58,14 +58,31 @@ export class TokenClient extends Core {
...
         CryptoEngine: Class<KeyStoreCryptoEngine>,
-        realmId?: string
+        realmId?: string,
+        tppType?: Number
     ): Promise<Member> {
-        return super.createMemberCore(alias, CryptoEngine, Member, 'BUSINESS', undefined, realmId);
+        let memberType = 'BUSINESS';
+        if (typeof tppType !== 'undefined' ) {
+            switch(tppType) {
+            case 2:
```
Comment (nikanka-token) [inline tpp/src/main/TokenClient.js:73]: I would not use "Type" name in SDK. It is internal terminology. The external one is reflected in lib-proto CreateMemberType.
We discussed public naming for quite a long time with the product team and we should use this official naming everywhere: 
T1 = BUSINESS - a customer of Token. Token is the regulated TPP in this case
T2 = LICENSED_TPP - a customer of Token but uses its own license. Token is a TSP for the business in this case.

**Comment by nikanka-token** [comment]
> @nikanka-token , after @yuhanniu 's comment on PR, I decided to not expose tppSdk and let it to create member of any type, but tpp type. So during creating member, you can use number to choose which type of tpp you want. Currently I only implemented type 2. Let me know if that part of logic is wrong.

I see, I would not use number for a type, as I mentioned in the other comment. I did not see Yuhan's comment unfortunately, but I think you can just use enum specific for tpp sdk with two values 'BUSINESS' and 'LICENSED_TPP' to show that only these two are available and defalt to 'BUSINESS'. Just do not use numbers please.

File: core/src/types.js [hunk 2]
```diff
@@ -49,6 +49,9 @@ export type EventType = 'INVALID'
     | 'TRANSFER_STATUS_CHANGED'
     | 'BULK_TRANSFER_STATUS_CHANGED';
 
+export type MemberType = 'BUSINESS'
```
Comment (nikanka-token) [inline core/src/types.js:52]: Does this actually restrict the MemberType to only BUSINESS or LICENSED_TPP in JS? Or it is just informative?

Comment (javidang) [reply core/src/types.js:52]: it will fail if the call is different value than these. This is not exactly enum, but only way to limit.

Comment (nikanka-token) [reply core/src/types.js:52]: Consider adding comments explaining what do these types mean

**Pull request merged**